### PR TITLE
Convert jets3t tests to javaSDK.

### PR DESCRIPTION
**Pull request by LaurenSpiegel**
This PR changes the unused jets3t functional testing code to use the AWS Java SDK since (1) the AWS Java SDK is commonly used and (2) is one of the few clients that use streaming v4 auth on puts.  

This PR also adds streaming v4 auth tests by making a number of puts of objects of different size.

### CODE REVIEW

File: tests/functional/jaws/src/test/java/com/scality/JavaTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	protected static AmazonS3 s3client;
+	public AmazonS3 getS3Client() { return this.s3client; }
+	public static final String bucketName = "somebucket";
+
+        //run once before all the tests
+	@BeforeClass public static void initConfig() throws Exception {
+		JSONParser parser = new JSONParser();
+		String path = Paths.get("../config.json").toAbsolutePath().toString();
+		System.out.println(path);
+    	JSONObject obj = (JSONObject) parser.parse(new FileReader(path));
```
Comment (GuillaumeGomez) [inline tests/functional/jaws/src/test/java/com/scality/JavaTest.java]: Why is the indent different starting from here?

File: tests/functional/jaws/src/test/java/com/scality/JavaTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    	JavaTest.accessKey = (String) obj.get("accessKey");
+    	JavaTest.secretKey = (String) obj.get("secretKey");
+	    JavaTest.transport = (String) obj.get("transport");
```
Comment (GuillaumeGomez) [inline tests/functional/jaws/src/test/java/com/scality/JavaTest.java]: Mix of tabs and spaces.

File: tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+import com.amazonaws.services.s3.model.PutObjectRequest;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.model.S3Object;
+
+
+@RunWith(Parameterized.class)
+public class StreamV4AuthTest {
+    private Integer fileSize;
+
+    protected static String accessKey;
```
Comment (GuillaumeGomez) [inline tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java]: Spaces indent.

File: tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+	public static  String getAccessKey() { return accessKey; }
```
Comment (GuillaumeGomez) [inline tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java]: Tab indent. In this file, it seems that spaces indent is the most present.

File: tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+	protected static String ipAddress;
+	public static String getIpAddress() { return ipAddress; }
+	protected static AmazonS3 s3client;
+	public static AmazonS3 getS3Client() { return s3client; }
+	public static final String bucketName = "streambucket";
+    public static final String objName = "streamObject";
+    public static final String fileName = "obj.txt";
+
+   @BeforeClass
+   public static void initialize() throws Exception {
```
Comment (GuillaumeGomez) [inline tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java]: Indent of 3. A bit weird.

File: tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+       JSONParser parser = new JSONParser();
```
Comment (GuillaumeGomez) [inline tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java]: And now indent of 4. :-/

**Comment by LaurenSpiegel** [comment]
@ironman-machine if you thought ruby was fun
SCALITY_INTEGRATION_BRANCH=test/javaSDK-lhs
SCALITY_CI_AGGRESSOR_FORCE_BUILD=true

**Comment by LaurenSpiegel** [comment]
@ironman-machine go the distance
SCALITY_INTEGRATION_BRANCH=test/javaSDK-lhs
SCALITY_CI_AGGRESSOR_FORCE_BUILD=true

**Comment by LaurenSpiegel** [comment]
@ironman-machine go the distance with the correct docker image this time
SCALITY_INTEGRATION_BRANCH=test/javaSDK-lhs
CI_AGGRESSOR_FORCE_BUILD=1

**Comment by LaurenSpiegel** [comment]
@ironman-machine https...
SCALITY_INTEGRATION_BRANCH=test/javaSDK-lhs
CI_AGGRESSOR_FORCE_BUILD=1

File: tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+    public static final String bucketName = "streambucket";
+    public static final String objName = "streamObject";
+    public static final String fileName = "obj.txt";
+
+    @BeforeClass
+    public static void initialize() throws Exception {
+        System.setProperty(SDKGlobalConfiguration.DISABLE_CERT_CHECKING_SYSTEM_PROPERTY, "true");
+        JSONParser parser = new JSONParser();
+        String path = Paths.get("../config.json").toAbsolutePath().toString();
+        System.out.println(path);
```
Comment (msegura) [inline tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java]: perhaps this System.out.println should be remove.
Or the message could be improve to make it clear that this is the path of the configuration file being used.
This is not a blocker in my opinion however.

Comment (LaurenSpiegel) [inline tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java]: removed.  thanks!

**Pull request merged**